### PR TITLE
Fix the Debug keyword

### LINKED ISSUE #114: Debug keyword fails: 'PASS' is not defined
With a Plone 5.2 build, when I insert the Debug keyword I get the following error:

 ```Evaluating expression 'PASS == 'PASS'' failed: NameError: name 'PASS' is not defined```

If I change the following line:
https://github.com/plone/plone.app.robotframework/blob/021cd5e7c785845c17afccadccbe018c0b5cc388/src/plone/app/robotframework/keywords.robot#L35
to ```Run keyword if  '${debug}[0]' == 'PASS'``` then the Debug keyword works correctly.

I noticed that in http://robotframework.org/robotframework/latest/libraries/BuiltIn.html#Evaluating%20expressions it says "Most importantly, strings must always be quoted" and gives the example: ```'${status}' == 'PASS'```, so perhaps this is a valid fix. However, if I evaluate the expression, it looks like it already returns 'PASS':
```
> Run keyword and ignore error  Import library  DebugLibrary  WITH NAME  DebugLibrary
< ('PASS', None)
```

Versions:
plone.app.robotframework-1.5.2
robotframework_debuglibrary-1.1.4
robotsuite-2.0.0
robotframework-3.1.2

### PR DESCRIPTION
**Pull request by cillianderoiste**
Fixes #114

### CODE REVIEW

**Comment by ale-rt** [comment]
@jenkins-plone-org please run jobs

File: src/plone/app/robotframework/keywords.robot [hunk 1]
```diff
@@ -32,11 +32,11 @@ Debug
     ${fallback} =  Run keyword and ignore error
     ...            Import library  plone.app.robotframework.keywords.Debugging
     ...            WITH NAME  DebuggingLibrary
-    Run keyword if  ${debug}[0] == 'PASS'
+    Run keyword if  '${debug}[0]' == 'PASS'
     ...             DebugLibrary.Debug
-    Run keyword if  ${debug}[0] == 'FAIL' and ${dialogs}[0] == 'PASS'
+    Run keyword if  '${debug}[0]' == 'FAIL' and ${dialogs}[0] == 'PASS'
```
Comment (thet) [inline src/plone/app/robotframework/keywords.robot]: @cillianderoiste does ``${dialogs}[0]`` also need to be quoted?

File: src/plone/app/robotframework/keywords.robot [hunk 2]
```diff
@@ -32,11 +32,11 @@ Debug
...
     ...             DialogsLibrary.Pause Execution
-    Run keyword if  ${debug}[0] == 'FAIL' and ${dialogs}[0] == 'FAIL'
+    Run keyword if  '${debug}[0]' == 'FAIL' and ${dialogs}[0] == 'FAIL'
```
Comment (thet) [inline src/plone/app/robotframework/keywords.robot]: @cillianderoiste also here...

**Comment by cillianderoiste** [comment]
Thanks @thet, I verified that the quotes around ${dialogs}[0] are also necessary, and have added them.

**Pull request merged**